packages/web-components/src/common/styles/theme.ts

```diff
@@ -24,6 +24,8 @@ export interface InitializerTheme {
 }
 
 export interface ThemeClasses {
+  searchWidgetAdvanceSeachLabel: string
```

Comment:
Can these keys be made more generic please so they can be re-used.

Suggest;

`featureButton` and `featureLabel` so can be re-used across components without binding their use semantically to the search widget.